src/GafferUI/StandardStyle.cpp

```diff
@@ -930,42 +934,53 @@ Imath::Color3f StandardStyle::colorForState( Color c, State s, const Imath::Colo
 
 static const std::string &vertexSource()
 {
+	// When isCurve is set, this renders a curve defined by start and end points, and start and end tangents.
+	// See noodleShapes.svg for explanation.  TODO - decide where this lives
```

Comment:
Hmm, yeah, we don't really have a defined place for this sort of thing do we? Maybe in contrib? Or what do you think about adding it to the wiki?